script/Ciel/Day 11/11_01_CIEL11_1.txt

```diff
@@ -6,22 +6,23 @@ A hot summer day.
 [sha:54a68f2f4f392134a075af0306c6c302423563f3]{
 -- Page 1, Offset 28972.
 -- 青い空と　大きな大きな入道雲。
-A blue sky with huge, towering cumulonimbus clouds.
+The blue sky.
+ Large, giant clouds towering above.
```

Comment:
I'm not necessarily against simplifying the language here, but "Large, giant" doesn't really work.

Maybe just "Large, large clouds towering above"

Reply:
Prefer original tbh